src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -238,6 +270,18 @@ private void ensureSelectedPersonIsValid(ListChangeListener.Change<? extends Ent
         }
     }
 
+    /**
+     * Ensures that storage is updated whenever entry book is modified.
+     */
+    private void ensureStorageSaved(Observable observable) {
```

Comment:
Maybe name this function ```saveToStorage``` instead? Make it sound more like an action. "ensure" sounds like a promise, even though an exception might be thrown

Reply:
I didn't name it that is because it has the signature of an InvalidationListener (Observable -> void), but perhaps a different name would work.

I think I'll go with `saveToStorageListener`

Reply:
icic, okays!